02/app.js

```diff
@@ -0,0 +1,33 @@
+const formEl = document.querySelector('form');
+formEl.addEventListener('submit', checkData);
+
+function checkData(e) {
+    e.preventDefault();
+    const email = e.target.elements.login.value;
+    const pass1 = e.target.elements.pass1.value;
+    const pass2 = e.target.elements.pass2.value;
+    const regulAccept = e.target.elements['accept'];
+    const errors = [];
+
+    if((email.includes('@')) && (pass1 === pass2) && (pass1.length>6) && (regulAccept.checked)){
+        console.log('done');
+    }
```

Comment:
Zamiast sprawdzać te warunku 2 razy (bo robisz to teraz i poniżej tylko z zaprzeczeniem).
To lepiej zrobić jeden warunek u dołu tj. `errors.length === 0` czyli jeśli nie dodano żadnego błędu to znaczy, że jest ok :)